oozie/src/test/java/org/apache/falcon/oozie/feed/OozieFeedWorkflowBuilderTest.java

```diff
@@ -99,6 +102,7 @@
     private static final String SRC_CLUSTER_PATH = "/feed/src-cluster.xml";
     private static final String TRG_CLUSTER_PATH = "/feed/trg-cluster.xml";
     private static final String FEED = "/feed/feed.xml";
+    private static final String POST_FEED = "/feed/fs-post-processing-feed.xml";
```

Comment:
Why create one more test file? FEED has both retention and replication that can be used in your test case, right?